DigitalOffice/Tests/UserServiceTests/ValidatorsTests/UserCredentialsValidatorTests.cs

```diff
@@ -0,0 +1,88 @@
+﻿using FluentValidation;
+using NUnit.Framework;
+using UserService.Database.Entities;
+using UserService.Models;
+using UserService.Validators;
+
+namespace UserServiceUnitTests.ValidatorsTests
+{
+    class UserCredentialsValidatorTests
+    {
+        public class Tests
```

Comment:
why class Tests in class UserCredentialsValidatorTests ??

Reply:
fixed